deploy/fb-publisher-chart/templates/deployment.yaml

```diff
@@ -35,8 +35,11 @@ spec:
           - configMapRef:
               name: fb-publisher-env-{{ .Values.environmentName }}
         env:
-          # external secrets
-          # defined in secrets.yml
+          - name: DATABASE_URL
+            valueFrom:
+              secretKeyRef:
+                name: rds-instance-formbuilder-publisher-{{ .Values.environmentName }}
+                key: url
```

Comment:
if you put this in the config map would not have to repeat yourself?

Reply:
how do i add to config map if it's from a secret?